test/companies/companies_test.exs

```diff
@@ -13,69 +13,10 @@ defmodule Companies.CompaniesTest do
-  #    end
-
-  #    test "trims leading and trailing whitespace on text search" do
-  #      insert(:company, name: "ZULU")
-  #      insert(:company, name: "BETA")
-  #      insert(:company, name: "ALPHA")
-  #
-  #      assert %{entries: [%{name: "ALPHA"}]} = Companies.all(%{"search" => %{"text" => "alpha "}})
-  #      assert %{entries: [%{name: "ALPHA"}]} = Companies.all(%{"search" => %{"text" => " alpha"}})
-  #      assert %{entries: [%{name: "ALPHA"}]} = Companies.all(%{"search" => %{"text" => " alpha "}})
-  #    end
-
-  #    test "filters companies by text (by location)" do
-  #      insert(:company, name: "ZULU", location: "Australia")
-  #      insert(:company, name: "BETA", location: "Brazil")
-  #      insert(:company, name: "ALPHA", location: "Australia")
-  #
-  #      assert %{entries: [%{name: "ALPHA"}, %{name: "ZULU"}]} = Companies.all(%{"search" => %{"text" => "aust"}})
-  #      assert %{entries: [%{name: "BETA"}]} = Companies.all(%{"search" => %{"text" => "zil"}})
-  #      assert %{entries: []} = Companies.all(%{"search" => %{"text" => "cana"}})
-  #    end
-
-  #    test "trims leading and trailing whitespace on text search (by location)" do
-  #      insert(:company, name: "ZULU", location: "Canada")
-  #      insert(:company, name: "BETA", location: "Brazil")
-  #      insert(:company, name: "ALPHA", location: "Australia")
-  #
-  #      assert %{entries: [%{name: "ALPHA"}]} = Companies.all(%{"search" => %{"text" => "aus "}})
-  #      assert %{entries: [%{name: "ALPHA"}]} = Companies.all(%{"search" => %{"text" => " australia"}})
-  #      assert %{entries: [%{name: "ALPHA"}]} = Companies.all(%{"search" => %{"text" => " lia "}})
-  #    end
-  #
-  #    test "filters companies by text (by name and location)" do
-  #      insert(:company, name: "AUSTO", location: "Canada")
-  #      insert(:company, name: "ALPHA", location: "Australia")
-  #
-  #      assert %{entries: [%{name: "ALPHA"}, %{name: "AUSTO"}]} = Companies.all(%{"search" => %{"text" => "aust"}})
-  #    end
-  #
-  #    test "trims leading and trailing whitespace on text search (by name and location)" do
-  #      insert(:company, name: "AUSTO", location: "Canada")
-  #      insert(:company, name: "ALPHA", location: "Australia")
-  #
-  #      assert %{entries: [%{name: "ALPHA"}, %{name: "AUSTO"}]} = Companies.all(%{"search" => %{"text" => "aust "}})
-  #      assert %{entries: [%{name: "ALPHA"}, %{name: "AUSTO"}]} = Companies.all(%{"search" => %{"text" => " aust"}})
-  #      assert %{entries: [%{name: "ALPHA"}, %{name: "AUSTO"}]} = Companies.all(%{"search" => %{"text" => " aust "}})
-  #    end
-  #  end
-
   describe "recent/1" do
     test "retrieves a paginated list of companies sorted by inserted datetime (desc)" do
       assert %{entries: entries} = Companies.recent()
-      assert [%{name: "ALPHA"}, %{name: "ZULU"}] = entries
+      assert [%{name: "ZULU"}, %{name: "ALPHA"}] = entries
```

Comment:
The A-Z sort was _actually_ at play here. Change the fixture dates to be reversed so that we could _really_ ensure this was working as expected.